File: app/src/main/java/br/com/levimendesestudos/spidermagazine/utils/ToastUtil.java
package br.com.levimendesestudos.spidermagazine.utils;

import android.content.Context;
import android.widget.Toast;

/**
 * Created by 809778 on 22/12/2016.
 */

public class ToastUtil {

    public static void showToast(Context context, String msg) {
        Toast.makeText(context, msg, Toast.LENGTH_LONG).show();
    }

    public static void showToast(Context context, int idMsg) {
        Toast.makeText(context, idMsg, Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/br/com/levimendesestudos/spidermagazine/utils/ToastUtil.kt
package br.com.levimendesestudos.spidermagazine.utils

import android.content.Context
import android.widget.Toast

/**
 * Created by 809778 on 22/12/2016.
 */

object ToastUtil {

    fun showToast(context: Context, msg: String) {
        Toast.makeText(context, msg, Toast.LENGTH_LONG).show()
    }

    fun showToast(context: Context, idMsg: Int) {
        Toast.makeText(context, idMsg, Toast.LENGTH_LONG).show()
    }
}
